src/evict/evict_page.c

```diff
@@ -248,7 +248,7 @@ __evict_page_dirty_update(WT_SESSION_IMPL *session, WT_REF *ref, bool closing)
 	switch (mod->rec_result) {
 	case WT_PM_REC_EMPTY:				/* Page is empty */
 		/* Discard the parent's address. */
-		__wt_ref_free_addr(session, ref);
+		__wt_ref_addr_free(session, ref);
```

Comment:
Should this be `__wt_ref_block_free`?  If not, how do we end up here with `ref->addr != NULL`?